src/TortoiseProc/Settings/SetLookAndFeelPage.cpp

```diff
@@ -32,23 +32,30 @@ extern MenuInfo menuInfo[];
 void InsertMenuItemToList(CListCtrl *list,CImageList *imagelist)
 {
 	int i=0;
+	HINSTANCE hRes = AfxGetResourceHandle();
 	while(menuInfo[i].command != ShellMenuLastEntry)
 	{
-		if ((menuInfo[i].command != ShellSeparator &&
-		   menuInfo[i].command != ShellSubMenu &&
-		   menuInfo[i].command != ShellSubMenuFile &&
-		   menuInfo[i].command != ShellSubMenuFolder &&
-		   menuInfo[i].command != ShellSubMenuLink &&
-		   menuInfo[i].command != ShellSubMenuMultiple) &&
-		   (i == 0 || menuInfo[i - 1].menuID != menuInfo[i].menuID))
+		const MenuInfo& mi = menuInfo[i];
+		if ((mi.command != ShellSeparator &&
+			mi.command != ShellSubMenu &&

```

Comment:
This can also be a separate commit with a message like "Use a temp variable instead of repeatedly calling the reference".